DOCKER.md

```diff
@@ -40,3 +59,15 @@ Then, set the `DEBUG` environment variable and expose the port where to connect
 ```bash
 docker run --rm -e DEBUG=true -p 5005:5005 -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
 ```
+
+## Custom War Packager
+There is an alternative to Vanilla Dockerfile that gives a huge versatility when it comes to generate the docker images.
+See the demo for a better understanding on [how to build Jenkinsfile Runner with Custom WAR Packager](demo/cwp).
+
+Once the docker image is generated you can execute it in the same way as a Vanilla image with the only exception of the master workspace.
```

Comment:
Why is the workspace an exception? The vanilla image works the same way.

Reply:
- Entrypoint in Vanilla image:
```
ENTRYPOINT ["/app/bin/jenkinsfile-runner-launcher", \
            "-w", "/app/jenkins",\
            "-p", "/usr/share/jenkins/ref/plugins",\
            "-f", "/workspace", \
            "--runWorkspace", "/build"]
```
- Entrypoint in Dockerfile generated by CWP:
```
ENTRYPOINT ["/app/bin/jenkinsfile-runner", \
            "-w", "/app/jenkins",\
            "-p", "/usr/share/jenkins/ref/plugins",\
            "-f", "/workspace/Jenkinsfile"]
```
The `runWorkspace` is not set by default to `/build` and that's why there is an exception and the next sentence is needed:

> If you are interested in mapping a volume and having access to it, both the -v docker option and the --runworkspace options needs to be specified.

Reply:
See https://github.com/jenkinsci/custom-war-packager/pull/71/files#diff-07df933f2a2d07a256ae752fa9ac363fR58

Reply:
Yes, needs fix